front/src/applications/stdcmV2/components/StdcmOrigin.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { useSelector } from 'react-redux';
+
+import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
+import { useOsrdConfSelectors, useOsrdConfActions } from 'common/osrdContext';
+import StdcmAllowances from 'modules/stdcmAllowances/components/StdcmAllowances';
+import type { StdcmConfSliceActions } from 'reducers/osrdconf/stdcmConf';
+import { useAppDispatch } from 'store';
+
+import StdcmCard from './StdcmCard';
+import StdcmOperationalPoint from './StdcmOperationalPoint';
+
+const StdcmOrigin = ({ isPending = false }: { isPending?: boolean }) => {
+  const { getOriginV2, getOriginDate, getOriginTime } = useOsrdConfSelectors();
+  const { updateOriginV2, updateOriginDate, updateOriginTime } =
+    useOsrdConfActions() as StdcmConfSliceActions;
+  const origin = useSelector(getOriginV2);
+  const originDate = useSelector(getOriginDate);
+  const originTime = useSelector(getOriginTime);
+  const dispatch = useAppDispatch();
+  return (
+    <StdcmCard name="Origine" disabled={isPending} hasTip>
+      <div className="stdcm-v2-origin">
+        <StdcmOperationalPoint updatePoint={updateOriginV2} point={origin} isPending={isPending} />
+        <div className="stdcm-v2-origin__parameters d-flex">
+          <div className="col-6">
+            <InputSNCF
+              id="dateOrigin"
+              label="Date"
+              type="date"
+              name="dateOrigin"
+              onChange={(e) => dispatch(updateOriginDate(e.target.value))}
+              value={originDate}
+              disabled={isPending}
+            />
+          </div>
+          <div className="col-6">
+            <InputSNCF
+              type="time"
+              label="Heure"
+              id="originTime"
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                dispatch(updateOriginTime(e.target.value))
+              }
+              value={originTime}
+              disabled={isPending}
+            />
+          </div>
+        </div>
```

Comment:
![Capture d’écran 2024-05-23 à 15 30 29](https://github.com/OpenRailAssociation/osrd/assets/113104137/5ac8d19f-0039-440c-9e42-39dd8742f8b0)

First and second rows are not aligned